### PR TITLE
Warn that translations will be cleared if the email is cleared when editing a status

### PR DESCRIPTION
**Pull request by clouser2**
### Description

In addition to the warning, I:
  * Repointed a TODO for SVG resizing at a more appropriate tech debt issue (#3148)
  * Removed a TODO that isn't necessary (clearing state can be achieved with a page refresh)

Screenshot:
Screen Shot 2022-08-12 at 2 42 02 PM

### Testing notes:

Status tracking browser tests are temporarily skipped until feature flags can be controlled within browser tests. I temporarily allowed CI to run with the tests enabled, which was successful (https://github.com/civiform/civiform/runs/7814428678?check_suite_focus=true).

## Release notes:

N/A

### Checklist

- [X] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [X] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

#2752; #3148

### CODE REVIEW

File: browser-test/src/support/admin_program_statuses.ts [hunk 1]
```diff
@@ -141,6 +141,22 @@ export class AdminProgramStatuses {
...
+    await this.page.click(
+      this.programStatusItemSelector(statusName) + ' button:has-text("Edit")',
+    )
+
+    const modal = await waitForAnyModal(this.page)
+    const innerText = await modal.innerText()
+    expect(innerText).toContain('Edit this status')
+
+    // Close the modal prior to any assertions to avoid affecting
+    // subsequent tests.
```
Comment (shanemc-goog) [inline browser-test/src/support/admin_program_statuses.ts:154]: wrap up

File: server/app/views/admin/programs/ProgramStatusesView.java [hunk 2]
```diff
@@ -321,9 +333,8 @@ private Modal makeStatusUpdateModal(
       Http.Request request,
       ProgramDefinition program,
       Form<ProgramStatusesForm> form,
-      boolean displayOnLoad) {
-    // TODO(#2752): If an email is already configured, add a warning that setting it to empty
-    // will clear any other localized text.
+      boolean displayOnLoad,
+      Optional<DomContent> extraContent) {
```
Comment (shanemc-goog) [inline server/app/views/admin/programs/ProgramStatusesView.java:337]: `extraContent` This feels unnecessarily vague, If we make it `boolean showEmailDeletionWarning` it adds clarity and readability.

**Review by shanemc-goog — CHANGES_REQUESTED** [review]

File: browser-test/src/support/admin_program_statuses.ts (hunk 1 above)
Comment (clouser2) [reply browser-test/src/support/admin_program_statuses.ts]: Done.

File: server/app/views/admin/programs/ProgramStatusesView.java (hunk 2 above)
Comment (clouser2) [reply server/app/views/admin/programs/ProgramStatusesView.java]: Done. One thing to note is that adding a second boolean has a different tradeoff in that it increases the risk of mixing variable order up. That said, I think introducing an AutoValue for this is too heavyweight so I'm not going to pursue that approach.

File: server/app/views/admin/programs/ProgramStatusesView.java [hunk 3]
```diff
@@ -226,12 +225,13 @@ private Pair<DivTag, ImmutableList<Modal>> renderStatusItem(
...
-            ? Optional.of(renderEmailTranslationWarning())
-            : Optional.empty();
     Modal editStatusModal =
-        makeStatusUpdateModal(request, program, statusEditForm, displayOnLoad, extraContent);
+        makeStatusUpdateModal(
+            request,
+            program,
+            statusEditForm,
+            displayOnLoad,
+            status.localizedEmailBodyText().isPresent());
```
Comment (shanemc-goog) [inline server/app/views/admin/programs/ProgramStatusesView.java:234]: prefix comment this too for readability as the evaluation doesn't map to the params utility.

**Review by shanemc-goog — CHANGES_REQUESTED** [review]

File: server/app/views/admin/programs/ProgramStatusesView.java (hunk 2 above)
Comment (shanemc-goog) [reply server/app/views/admin/programs/ProgramStatusesView.java]: ack. That's not unique here though and there are better ways to address it than what we had if it's of concern.   

If the method can be made static we could use AutoBuilder!!

**Review by shanemc-goog — APPROVED** [review]
LGTM, Just the one comment

File: server/app/views/admin/programs/ProgramStatusesView.java (hunk 3 above)
Comment (clouser2) [reply server/app/views/admin/programs/ProgramStatusesView.java]: Done.

**Comment by clouser2** [comment]
After talking with @rahaghas , we decided to update the language to:

Please be aware that clearing the email body will also clear any associated translations

I'll send out a follow-on PR for that shortly.

**Pull request merged**